tests/NLog.UnitTests/LayoutRenderers/StackTraceRendererTests.cs

```diff
@@ -118,6 +118,27 @@ public void RenderStackTrace_DetailedFlat()
             AssertDebugLastMessageContains("debug", " => [Void RenderStackTrace_DetailedFlat()] => [Void RenderMe(System.String)]");
         }
 
+#if MONO
+        [Fact(Skip = "MONO not supporting StackTrace with source")]
+#else
+        [Fact]
+#endif
+        public void RenderStackTrace_RawWithSource()
+        {
+            LogManager.Configuration = CreateConfigurationFromString(@"
+            <nlog>
+                <targets><target name='debug' type='Debug' layout='${message} ${stacktrace:format=RawWithSource}' /></targets>
+                <rules>
+                    <logger name='*' minlevel='Debug' writeTo='debug' />
+                </rules>
+            </nlog>");
+
+            RenderMe("I am:");
+
+            AssertDebugLastMessageContains("debug", "RenderStackTrace_RawWithSource at offset ");
```

Comment:
"Allow edits from maintainers" is checked, so I guess you can knock yourself wild :)